.rat-excludes

```diff
@@ -54,3 +54,4 @@ build/scala-*/**
 **/BeeLine.properties
 **/sql-keywords.properties
 **/python/**
+**/*.py
```

Comment:
we also have other py source files in project, so we can ignore all the py files

Reply:
I do ignore all the pyfiles here, as rat checks may not be suitable for python scripts?
Do you mean 'we cannnot'  ignore all the pyfiles?

Reply:
yes, we can not

Reply:
OK. Removed this exclusion for python scripts.